### DISCUSSION TITLE
Prism mock - Support multiple files as input

### DISCUSSION THREAD

**Discussion by DeepthiDinesh** [stoplightio/prism#2003] [Q&A]
With the current set up we are able to pass only a single Swagger or Open API file as input to generate mock responses and start the mock server. Is it possible to provide multiple files in and prism can generate mock responses for multiple files?  

prism mock <document>

**Comment by paul-uz** [comment]
Any update on this?

I need to be able to provide mocks for a large number of individual schema files
